### PR TITLE
Release/0.3.5

### PR DESCRIPTION
**Pull request by 4ydan**
## Basics

- [ ] I added a line to [changelog.md](/doc/changelog.md)
- [ ] The PR is rebased with current master.
- [ ] Details of what you changed are in commit messages.
- [ ] References to issues, e.g. `close #X`, are in the commit messages and changelog.
- [ ] The buildserver is happy.

## Checklist

- [ ] I have installed and I am using [pre-commit hooks](../doc/contrib/README.md#Hooks)
- [ ] I fully described what my PR does in the documentation
- [ ] I fixed all affected documentation
- [ ] I fixed the introduction tour
- [ ] I wrote migrations in a way that they are compatible with already present data
- [ ] I fixed all affected decisions
- [ ] I added automated tests or a [manual test protocol](../doc/tests/manual/protocol.md)
- [ ] I added code comments, logging, and assertions as appropriate
- [ ] I translated all strings visible to the user
- [ ] I mentioned [every code or binary](https://github.com/ElektraInitiative/PermaplanT/blob/master/.reuse/dep5) not directly written or done by me in [reuse syntax](https://reuse.software/)
- [ ] I created left-over issues for things that are still to be done
- [ ] Code is conforming to [our Architecture](/doc/architecture)
- [ ] Code is conforming to [our Guidelines](/doc/guidelines)
- [ ] Code is consistent to [our Design Decisions](/doc/decisions)
- [ ] Exceptions to any guidelines are documented

## Review

- [ ] I've tested the code
- [ ] I've read through the whole code
- [ ] I've read through the whole documentation
- [ ] I've checked conformity to guidelines

### CODE REVIEW

**Comment by 4ydan** [comment]
We can permanently enable the external link checker now, I put the links that are causing problems to the exclude config.

**Comment by markus2330** [comment]
> We could permanently enable the link checker, I added the problematic links to the exlude config. But now jenkins has some TLS problems:

Yes, because of that it is probably still better to only run it manually.

File: doc/changelog.md [hunk 1]
```diff
@@ -6,35 +6,78 @@ Use a not-yet-used-in-any-PR random line in the top-most version.
 
 Syntax: `- short text describing the change _(Your Name)_`
 
-## 0.3.5 - UNRELEASED
+## 0.3.6 - UNRELEASE
```
Comment (markus2330) [inline doc/changelog.md:9]: This PR is a bit of a mixture of things that need to be taken in the release (fix of links, but I think it was already fixed by @tschawax), could be taken in (Cargo.toml changes) and changes that cannot be taken in (pumping of the version number to the next version).
Erlaa
```suggestion
## 0.3.6 - UNRELEASED
```

**Review by markus2330 — APPROVED** [review]
Great job! There is a merge conflict now, though.

**Comment by markus2330** [comment]
You can merge and release this. :rocket:

**Pull request merged**